src/main/java/com/google/javascript/cl2dts/DeclarationGenerator.java

```diff
@@ -150,6 +163,35 @@ public static void main(String[] args) {
     System.exit(0);
   }
 
+  List<String> parseDepgraphRoots() {
```

Comment:
No, it's strictly harder, sometimes I just ctrl-A and then paste into my cider window.
But I'll break it up now. I'll do the Options class and maybe some others that I've been thinking about.